controllers/storagecluster/reconcile.go

```diff
@@ -107,7 +107,7 @@ var validTopologyLabelKeys = []string{
 
 // +kubebuilder:rbac:groups=ocs.openshift.io,resources=*,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=ceph.rook.io,resources=cephclusters;cephblockpools;cephfilesystems;cephnfses;cephobjectstores;cephobjectstoreusers;cephrbdmirrors;cephblockpoolradosnamespaces,verbs=*
-// +kubebuilder:rbac:groups=noobaa.io,resources=noobaas,verbs=*
+// +kubebuilder:rbac:groups=noobaa.io,resources=noobaas,verbs=get;list;watch;create;update
```

Comment:
Have you tried uninstall, We may need delete as well.

Reply:
@iamniting I see this code now https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/noobaa_system_reconciler.go#L313